WeakAuras/Transmission.lua

```diff
@@ -1675,6 +1675,18 @@ function WeakAuras.ImportString(str)
   end
 end
 
+function crossRealmSendCommMessage(prefix, text, chattype, target, queueName, callbackFn, callbackArg)
```

Comment:
There's no point in having a chattype argument in here, as that is always "WHISPER".

Reply:
indeed